src/jazayeri_lab_to_nwb/watters/wattersnwbconverter.py

```diff
@@ -26,6 +30,10 @@ class WattersNWBConverter(NWBConverter):
     """Primary conversion class for my extracellular electrophysiology dataset."""
 
     data_interface_classes = dict(
+        RecordingVP0=WattersDatRecordingInterface,
+        SortingVP0=KiloSortSortingInterface,
+        RecordingVP1=WattersDatRecordingInterface,
+        SortingVP1=KiloSortSortingInterface,
         RecordingNP=SpikeGLXRecordingInterface,
         LFP=SpikeGLXRecordingInterface,
```

Comment:
Sorry for not catching this earlier; I'd rename this 'LF'

Will need to update all references elsewhere to this key

```suggestion
        LF=SpikeGLXRecordingInterface,
```